disqus/rest-api/class-disqus-rest-api.php

```diff
@@ -763,11 +763,9 @@ private function comment_data_from_post( $post ) {
             update_post_meta( $wp_post_id, 'dsq_thread_id', $thread['id'] );
         }
 
-        if ( null === $wp_post_id || false == $wp_post_id ) {
```

Comment:
Removing this check because it would prevent comments left via RSS from syncing.  Without this check, the comment still syncs in the WordPress database without a clickable link to the comment URL.